domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/FullAdmissionResponse.java

```diff
@@ -1,39 +1,17 @@
 package uk.gov.hmcts.cmc.domain.models.response;
 
 import lombok.Builder;
-import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
-import uk.gov.hmcts.cmc.domain.constraints.DateNotInThePast;
-import uk.gov.hmcts.cmc.domain.constraints.ValidFullAdmission;
 import uk.gov.hmcts.cmc.domain.models.PaymentOption;
 import uk.gov.hmcts.cmc.domain.models.RepaymentPlan;
 import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
 import uk.gov.hmcts.cmc.domain.models.party.Party;
 import uk.gov.hmcts.cmc.domain.models.statementofmeans.StatementOfMeans;
 
 import java.time.LocalDate;
-import java.util.Objects;
-import java.util.Optional;
-import javax.validation.Valid;
-import javax.validation.constraints.NotNull;
 
 import static uk.gov.hmcts.cmc.domain.models.response.ResponseType.FULL_ADMISSION;
-import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
-
-@ValidFullAdmission
-public class FullAdmissionResponse extends Response {
-
-    @NotNull
-    private final PaymentOption paymentOption;
-
-    @DateNotInThePast
-    private final LocalDate paymentDate;
-
-    @Valid
-    private final RepaymentPlan repaymentPlan;
-
-    @Valid
-    private final StatementOfMeans statementOfMeans;
 
+public class FullAdmissionResponse extends AdmissionResponse {
```

Comment:
Thoughts on composing this rather than inheriting it, given that AdmissionResponse is already extending from another class?
an interface could handle core fields on both just as well as inheritance but it would be more flexible.
@yarpo @sabahirfan ?